config/locales/en.yml

```diff
@@ -160,29 +160,28 @@ en:
         intake: Intake
         prep: Tax prep
         review: Quality review
-        finalize: Final steps
-        filed: Filing completed
+        file: Final steps
       status:
         intake_before_consent: Before consent
-        intake_in_progress: In progress
-        intake_open: Open
-        intake_review: In review
-        intake_more_info: Needs more information
+        intake_in_progress: Not ready
```

Comment:
I updated a lot of the enum names of the statuses to reflect the new statuses, but in the case of "in progress" I decided to leave it as "in_progress" for developer facing stuff -- it seemed really weird to have a negative in the name of the status, especially because its not a "bad" thing

Reply:
Good idea!